sample/db/samples/menu_items.rb

```diff
@@ -0,0 +1,240 @@
+main_menu = Spree::Menu.find_by!(name: 'Main Menu')
+
+##############
+# Root Items #
+##############
+main_menu_root_women = Spree::MenuItem.where(
+  name: 'Women',
+  item_type: 'Link',
+  menu_id: main_menu
+).first_or_create!
+
+main_menu_root_men = Spree::MenuItem.where(
+  name: 'Men',
+  item_type: 'Link',
+  menu_id: main_menu
+).first_or_create!
+
+main_menu_root_sw = Spree::MenuItem.where(
+  name: 'Sportsware',
+  item_type: 'Link',
+  menu_id: main_menu
+).first_or_create!
+
+##############
+# Catagories #
+##############
+main_menu_cat_women = Spree::MenuItem.where(
+  name: 'Catagories',
+  item_type: 'Container',
+  menu_id: main_menu,
+  parent_id: main_menu_root_women
+).first_or_create!
+
+main_menu_cat_men = Spree::MenuItem.where(
+  name: 'Catagories',
+  item_type: 'Container',
+  menu_id: main_menu,
+  parent_id: main_menu_root_men
+).first_or_create!
+
+main_menu_cat_sw = Spree::MenuItem.where(
+  name: 'Catagories',
+  item_type: 'Container',
+  menu_id: main_menu,
+  parent_id: main_menu_root_sw
+).first_or_create!
+
+##############
+# Promotions #
+##############
+main_menu_promo_women = Spree::MenuItem.where(
+  name: 'Promos',
+  item_type: 'Container',
+  menu_id: main_menu,
+  parent_id: main_menu_root_women
+).first_or_create!
+
+main_menu_promo_men = Spree::MenuItem.where(
+  name: 'Promos',
+  item_type: 'Container',
+  menu_id: main_menu,
+  parent_id: main_menu_root_men
+).first_or_create!
+
+main_menu_promo_sw = Spree::MenuItem.where(
+  name: 'Promos',
+  item_type: 'Container',
+  menu_id: main_menu,
+  parent_id: main_menu_root_sw
+).first_or_create!
+
+PROMOS = [main_menu_promo_women, main_menu_promo_men, main_menu_promo_sw]
+summer = Spree::Taxon.find_by!(permalink: 'new-collection/summer-2021')
+offers = Spree::Taxon.find_by!(permalink: 'special-offers/30-percent-off')
+
+#####################
+# Links For: PROMOS #
+#####################
+PROMOS.each do |promo|
+  summer_promo = Spree::MenuItem.where(
+    name: 'New Collection',
+    subtitle: "Summer #{Date.today.year}",
+    linked_resource_type: 'Taxon',
+    linked_resource_id: summer.id,
+    item_type: 'Promotion',
+    menu_id: main_menu,
+    parent_id: promo
+  ).first_or_create!
+
+  summer_promo.linked_resource_id = summer.id
+  summer_promo.save!
+
+  special_offer = Spree::MenuItem.where(
+    name: 'Special Offers',
+    subtitle: 'Get up to 30% OFF',
+    linked_resource_type: 'Taxon',
+    linked_resource_id: offers.id,
+    item_type: 'Promotion',
+    menu_id: main_menu,
+    parent_id: promo
+  ).first_or_create!
+
+  special_offer.linked_resource_id = offers.id
+  special_offer.save!
+end
+
+#################################
+# Links For: WOMEN / CATAGORIES #
+#################################
+Spree::MenuItem.where(
+  name: 'Skirts',
+  linked_resource_type: 'URL',
```

Comment:
@MatthewKennedy can we link to taxon please?

Reply:
Absolutely, I will give that a good going over today and get that sample data all correct.